Makefile

```diff
@@ -1,38 +1,44 @@
-commit  := $(shell git rev-parse --short HEAD)
-tag     := $(shell git tag -l 'v*-rc*' --points-at HEAD)
-version := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
-artifactory_publish := $(shell if [[ -n "$(tag)" ]]; then echo release; else echo dev; fi)
+artifact_name   := company-accounts-library
+version         := "unversioned"
 
 .PHONY: all
 all: build
 
 .PHONY: clean
 clean:
-	mvn clean
+    mvn clean
```

Comment:
These need to stay as tabs as it's a Makefile. Ordinarily yes we'd go the other way